File: reactClientApp/src/Components/Models/Requirement.js
export default class Requirement {
    constructor(
        _id,
        _idRequestPerson,
        _idTypeJob,
        _idRequirementStatus,
        date,
        address,
        description,
        zone) {

        this._id = _id;
        this._idRequestPerson = _idRequestPerson;
        this._idTypeJob = _idTypeJob;
        this._idRequirementStatus = _idRequirementStatus;
        this.date = date;
        this.address = address;
        this.description = description;
        this.zone = zone;

    }

}

File: src/main/java/com/example/apiFrontend/models/Requirement.java
package com.example.apiFrontend.models;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import org.springframework.lang.Nullable;

import java.util.Date;

@Data
@Getter
@Setter
public class Requirement {

    @Nullable
    private String _id;
    private String _idRequestPerson;
    private String _idTypeJob;
    private String _idRequirementStatus;
    private Date date;
    private Address address;
    private String description;
    private String zone;
    private boolean deleted;
}
